src/include/stir/listmode/CListRecordROOT.h

```diff
@@ -28,134 +28,45 @@
 #define __stir_listmode_CListRecordROOT_H__
 
 #include "stir/listmode/CListRecord.h"
-#include "stir/listmode/CListEventCylindricalScannerWithDiscreteDetectors.h"
+#include "stir/listmode/CListEventROOT.h"
+#include "stir/listmode/CListTimeROOT.h"
 #include "stir/Succeeded.h"
-#include "stir/round.h"
-#include "boost/static_assert.hpp"
-#include "stir/DetectionPositionPair.h"
-#include <boost/cstdint.hpp>
+#include "stir/Scanner.h"
+#include <boost/shared_ptr.hpp>
```

Comment:
cannot use boost/shared_ptr as we're moving to std::shared_ptr. in fact, we're probably not anyway as we don't have boost:: below. So remove this include